ui/storybook/README.md

```diff
@@ -0,0 +1,122 @@
+
+The project enforces the above contraints using the following tooling:
+
+- **aliases to source for all @perses-dev packages in storybook's webpack config** - E.g. `@perses-dev/components` will alias to `ui/components/src` instead of pointing to `ui/components/dist`). This makes it easy to import from packages in stories in a consistent way while pointing at the top level `src` export. It also ensures that components referenced indirectly in stories (usually a child of the component used directly in the story) point to source code to avoid accidentally mixing and matching source and compiled code, which can cause issues when using `context` and singletons.
+- **alias to source for the specific @perses-dev package in package-specific tsconfig** - E.g. a story in the `@perses-dev/components` package can import from `@perses-dev/components` and have TypeScript look in `ui/components/src` for types. Without this, you would get type errors because TypeScript will expect the package to have installed itself.
+
+## Notable addons
+
+Below are some notable addons this project currently uses.
+
+- [Accessibility](https://storybook.js.org/addons/@storybook/addon-a11y/) - Helpful for making components a11y.
+- [Actions](https://storybook.js.org/addons/@storybook/addon-actions) - Used to display data received by event handlers.
+- [Controls](https://storybook.js.org/addons/@storybook/addon-controls/) - Interact with component inputs dynamically in the Storybook UI.
+- [Dark Mode](https://storybook.js.org/addons/storybook-dark-mode/) - Adds a toolbar toggle that switches between light and dark mode.
+- [Docs](https://storybook.js.org/addons/@storybook/addon-docs/) - Document component usage and properties in Markdown.
+- [Links](https://storybook.js.org/addons/@storybook/addon-links/) - Use to create links that navigate between stories.
+- [Measure](https://storybook.js.org/addons/@storybook/addon-measure/) - Helpful for inspecting layouts by visualizing the box model.
+- [Outline](https://storybook.js.org/addons/@storybook/addon-outline/) - Outline all elements with CSS to help with layout placement and alignment.
+- [Storysource](https://storybook.js.org/addons/@storybook/addon-storysource/) - Used to show stories source in the addon panel
+- [Viewport](https://storybook.js.org/addons/@storybook/addon-viewport/) - Allows stories to be displayed in different sizes and layouts.
+
+We primarily use first party addons maintained by Storybook to avoid pain with upgrades and interoperability with the relatively complex Storybook ecosystem.
```

Comment:
Thoughts?

Reply:
Seems good, I don't know of any non-first-party-addons that I even need (seems like we have the things we need)